.github/workflows/ci.yml

```
@@ -162,10 +168,11 @@ jobs:
     with:
       clang-version: 20
       cmake-args: -DBUILD_MQT_QMAP_BINDINGS=ON
-      files-changed-only: true
+      files-changed-only: false
```

```suggestion
      files-changed-only: true
```
Just to not forget this.

---

Yes, thanks!